cpp/src/KIM_FilesystemPath.cpp

```diff
@@ -33,6 +33,7 @@
 
 #include <cstdlib>
 #include <iostream>
+#include <sstream>
```

Comment:
Why is this needed?  It seems to work without it.

Reply:
It's needed for [``std::basic_istringstream``](https://en.cppreference.com/w/cpp/io/basic_istringstream) referenced in ``PathList::Parse()``. Yeah, on most platforms you can get away without an explicit ``<sstream>`` include, but Linux/MinGW requires it.

Reply:
OK, please add something like `// For Linux/MinGW  IWYU pragma: keep`

Reply:
Done